BOJ/1000/hikang.py

```diff
@@ -0,0 +1,2 @@
+A, B = input().split()
+print(int(A) + int(B))
```

Comment:
```suggestion
A, B = map(int, input().split())
print(A + B)
```
`map()` 쓰면 int 두 번 쓰는 거 생략 가능